pkg/ibmcsidriver/controller_helper.go

```diff
@@ -329,8 +345,30 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 
 	//TODO port the code from VPC BLOCK to find region if zone is given
 
-	//If the zone is not provided in storage class parameters then we pick from the Topology
-	if len(strings.TrimSpace(volume.Az)) == 0 {
+	// validate bandwidth for dp2 profile
+	if volume.VPCVolume.Profile.Name == DP2Profile && volume.VPCVolume.Bandwidth > 0 {
+		err = fmt.Errorf("bandwidth is not supported for dp2 profile; please remove the property")
```

Comment:
pl change `please remove the property` to `please remove the property from storage class`